src/BuildGit.cpp

```diff
@@ -278,7 +278,8 @@ void buildGit(
 	std::string const & Prefix)
 {
 	// Initialize libgit2
-	checkRet(git_libgit2_init(), "git_libgit2_init()");
+	//checkRet(git_libgit2_init(), "git_libgit2_init()");
```

Comment:
What is wrong with `checkRet()` ?

Reply:
The input is returning 1 for some reason, I didn't investigate why yet

Reply:
Yeah its been annoying as hell